plasmapy/constants/__init__.py

```diff
@@ -34,3 +38,26 @@
 )
 
 from astropy.constants import atm
+
+# The following code is modified from astropy.constants to produce a
+# table containing information on the constants contained with PlasmaPy.
+# Mathematical constants can be just entered.
+
+_lines = [
+    'The following constants are available:\n',
+    '========== ================= ================ ============================================',
+    'Name       Value             Units            Description',
+    '========== ================= ================ ============================================',
+    "   pi      3.141592653589793                  Ratio of circumference to diameter of circle",
+]
+
+_constants = [eval(item) for item in dir() if item[0] != '_' and item != 'pi']
+for _const in _constants:
+    _lines.append('{0:^10} {1:^17.12g} {2:^16} {3}'
+                  .format(_const.abbrev, _const.value, _const._unit_string, _const.name))
+
+_lines.append(_lines[1])
+
+__doc__ += '\n'.join(_lines)
+
+del _lines, _const, _constants
```

Comment:
Oooooh. Clever!

Reply:
The credit should go to the Astropy developers who did this for `astropy.constants` in the first place. :slightly_smiling_face:

Reply:
Yeah, I saw the note. Still, it's clever. :D